src/contracts/service/CollabLandDAO.sol

```diff
@@ -3,10 +3,10 @@
 pragma solidity ^0.7.0;
 
 import "moloch/contracts/Moloch.sol";
-import "moloch/contracts/v3/core/DaoFactory.sol";
-import "moloch/contracts/v3/core/proposals/Proposal.sol";
-import "moloch/contracts/v3/core/membership/Member.sol";
-import "moloch/contracts/v3/adapters/Voting.sol";
+// import "moloch/contracts/v3/core/DaoFactory.sol";
+// import "moloch/contracts/v3/core/proposals/Proposal.sol";
+// import "moloch/contracts/v3/core/membership/Member.sol";
+// import "moloch/contracts/v3/adapters/Voting.sol";
```

Comment:
I added these import statements as a workaround to force typechain to generate corresponding TypeScript classes/interaces. I guess we can move it to a dummy contract.